lib/diagnostic.js

```diff
@@ -8,21 +8,40 @@
 // and returns the minimum from that set
 // assume that all the argument are numbers
 const min = function min() {
+  let lowNum = arguments[0];
 
+  for (let i = 0; i < arguments.length; i--) {
+    if (lowNum < arguments[i]) {
+      lowNum = arguments[i];
+    }
+  }
+
+  return lowNum;
 };
 
 // Question 2
 // Write a function that takes zero or more arguments
 // and returns the sum of that set
 // assume that all the argument are numbers
-const sum = function sum() {
+const sum = function sum(num1, num2) {
```

Comment:
This should have been left as is, since there might no, one, or more than two arguments.